app/controllers/api/v2/base_controller.rb

```diff
@@ -61,12 +61,12 @@ def root_node_name
       end
 
       def metadata_total
-        @total ||= resource_scope.try(:count).to_i
+        @total ||= resource_scope.try(:size).to_i
```

Comment:
I do not really mind either way, I am just curious if there is a particular reason for swapping `:size` for `:count`.

Reply:
if the scope has already been materialized, `size` will return the array length, while `count` will generate a new db `COUNT(*)` query. (https://mensfeld.pl/2014/09/activerecord-count-vs-length-vs-size-and-what-will-happen-if-you-use-it-the-way-you-shouldnt/)